src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -56,6 +56,9 @@
     <Compile Include="System\Net\Quic\Implementations\MsQuic\Interop\MsQuicStatusCodes.OSX.cs" />
   </ItemGroup>
   <!-- Project references -->
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+    <PackageReference Include="System.Net.MsQuic.Transport" PrivateAssets="all" Version="$(SystemNetMsQuicTransportVersion)" />
```

Comment:
@ViktorHofer is this the best way to consume this given that we want it to be part of the shared framework?

Reply:
Was discussing this with @wfurt offline. I think the PackageReference is fine but this isn't enough to binplace it into the runtime pack. I'm currently discussing what it takes to make binplace pick up the assembly as it is RID specific.